pkgs/by-name/gi/git-conventional-commits/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+let
+  pname = "git-conventional-commits";
+  version = "2.6.7";
+in
+buildNpmPackage {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "qoomon";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-A5J5h6TrjPmMDQCTAn6TKTA71uu5Qe/Si32cx57vdmM=";
+  };
+
+  npmDepsHash = "sha256-8kf0Wx2YTjZDswFaLJzbxf4Tpjd3h47msOvU+JSBpWE=";
+
+  dontNpmBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,share/${pname}}
+    cp cli.js git-conventional-commits.default.yaml commit-msg.sh $out/share/${pname}
+    cp -r lib/ node_modules/ $out/share/${pname}
+    ln -s $out/share/${pname}/cli.js $out/bin/${pname}
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    homepage = "https://github.com/qoomon/git-conventional-commits";
+    description = "Git Conventional Commits Util to generate Semantic Version and Markdown Change Log and Validate Commit Message";
```

Comment:
```suggestion
    description = "Generate semantic versions, markdown changelogs, and validate commit messages";
```

Reply:
Thank you! I have adopted your suggestion. However, I’m not sure about the style of description we should follow. Could you provide some guidance?

Reply:
See https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes

Reply:
@donovanglover tks! May I request your review again?